src/themes/oneui/base-variables/_sizing.scss

```diff
@@ -49,22 +31,22 @@ $stepper-button-size: 24px !default;
 
     --outline-size: #{$outline-size};
 
-    --space-0: #{$space-0};
-    --space-25: #{$space-25};
-    --space-50: #{$space-50};
-    --space-75: #{$space-75};
-    --space-100: #{$space-100};
-    --space-150: #{$space-150};
-    --space-200: #{$space-200};
-    --space-250: #{$space-250};
-    --space-300: #{$space-300};
-    --space-400: #{$space-400};
-    --space-500: #{$space-500};
-    --space-600: #{$space-600};
-    --space-800: #{$space-800};
-    --space-1000: #{$space-1000}; 
-
-    --margin: #{$margin};
+    --space-0: 0px;
+    --space-25: calc(0.25 * var(--space-100));
+    --space-50: calc(0.5 * var(--space-100));
+    --space-75: calc(0.75 * var(--space-100));
+    --space-100: 8px;
+    --space-150: calc(1.5 * var(--space-100));
+    --space-200: calc(2 * var(--space-100));
+    --space-250: calc(2.5 * var(--space-100));
+    --space-300: calc(3 * var(--space-100));
+    --space-400: calc(4 * var(--space-100));
+    --space-500: calc(5 * var(--space-100));
+    --space-600: calc(6 * var(--space-100));
+    --space-800: calc(8 * var(--space-100));
+    --space-1000: calc(10 * var(--space-100));
+
+    --margin: var(--space-25);
 
     --site-container-min-size: #{$site-container-min-size};
     --site-container-max-size: #{$site-container-max-size};
```

Comment:
can we define css directly for all variables, `--site-container-max-size`, `--site-container-max-size`, `--two-pane-left-width`, etc.?